docs/apache-airflow/concepts.rst

```diff
@@ -538,6 +538,41 @@ For ``smart sensor``, you need to configure it in ``airflow.cfg``, for example:
 For more information on how to configure ``smart-sensor`` and its architecture, see:
 :doc:`Smart Sensor Architecture and Configuration<smart-sensor>`
 
+Timeout:
```

Comment:
Make this a subheading so that it is linkable

Reply:
Thanks. That's done.